WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -209,6 +209,33 @@ decisions = {
 		}
 	}
 	
+	build_hagia_sophia = {
+		only_playable = yes
+		is_high_prio = yes
+		potential = {
+			religion_group = christian 
+			has_landed_title = c_byzantion
+			NOT = { has_global_flag = hagia_sophia_built }
+			NOT = { has_global_flag = building_hagia_sophia }
+		}
+		allow = {
+			wealth = 1500
+		}
+		effect = {
+			hidden_tooltip = {
+				character_event = { id = imperial_legacy.40 }
+				narrative_event = { id = imperial_legacy.41 days = 2 random = 50 }
```

Comment:
Isn't this a bit quick? I would make it into the historical 5 years and 10 months with the 50 random days.